src/operations/operation.ts

```diff
@@ -75,6 +75,7 @@ export abstract class AbstractOperation<TResult = any> {
     this.trySecondaryWrite = false;
   }
 
+  // Must match the first key of the command object sent to server
```

Comment:
```suggestion
  /** Must match the first key of the command object sent to the server */
```

This format makes it a doc comment, then it will appear when you hover over it in VSCode